app/routes/invitations.rb

```diff
@@ -0,0 +1,76 @@
+module ExercismWeb
+  module Routes
+    class Invitations < Core
+      post '/teams/:slug/invitation' do |slug|
+        please_login
+        only_for_team_managers(slug, "You are not allowed to add team members.") do |team|
+          team.invite_with_usernames(params[:usernames], current_user)
+          team.save
+
+          flash[:success] = "Invitation sent to team #{slug}"
+          redirect "/teams/#{slug}/manage"
+        end
+      end
+
+      post '/teams/:slug/invitation/accept' do |slug|
```

Comment:
Actually, in this case let's leave the `invitation` as singular. It's "Your invitation to this team".